### ISSUE TITLE
notifications showing complete name instead of the nick

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#90]
```
Pino's timeline does show the nick, but the notifications are still showing 
the complete name of the user.

```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 12 Feb 2010 at 9:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 12 Feb 2010 at 10:39
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 6643046b53 revision.
```

Original comment by `tro...@gmail.com` on 17 Feb 2010 at 7:30
- Changed state: **Fixed**
